.github/workflows/continuous-integration-workflow.yml

```diff
@@ -0,0 +1,16 @@
+name: Java CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.8

```

Comment:
i would suggest to bump to java version to 11.  create an issue and we can do it later via another PR